tutorials/3d/baked_lightmaps.rst

```diff
@@ -46,28 +54,36 @@ not share pixels in the texture.
 
 There are a few ways to ensure your object has a unique UV2 layer and texture size:
 
-Unwrap from your 3D DCC
-~~~~~~~~~~~~~~~~~~~~~~~
+Unwrap on scene import
+~~~~~~~~~~~~~~~~~~~~~~
 
-One option is to do it from your favorite 3D app. This approach is generally
-not recommended, but it's explained first so that you know it exists.
-The main advantage is that, on complex objects that you may want to re-import a
-lot, the texture generation process can be quite costly within Godot,
-so having it unwrapped before import can be faster.
+This is probably the best approach overall. The only downside is that, on large
+models, unwrapping can take a while on import. Nonetheless, Godot will cache the UV2
+across reimports, so it will only be regenerated when needed.
 
-Simply do an unwrap on the second UV2 layer.
+Select the imported scene in the filesystem dock, then go to the **Import** dock.
+There, the following option can be modified:
 
-.. image:: img/baked_light_blender.png
+.. image:: img/baked_light_import.png
 
-And import normally. Remember you will need to set the texture size on the mesh
-after import.
+The **Light Baking** mode needs to be set to **Gen Lightmaps**. A texel size
+in world units must also be provided, as this will determine the
+final size of the lightmap texture (and, in consequence, the UV padding in the map).
 
-.. image:: img/baked_light_lmsize.png
+The effect of setting this option is that all meshes within the scene will have
+their UV2 maps properly generated.
 
-If you use external meshes on import, the size will be kept.
-Be wary that most unwrappers in 3D DCCs are not quality oriented, as they are
-meant to work quickly. You will mostly need to use seams or other techniques to
-create better unwrapping.
+.. warning::
+
+    When reusing a mesh within a scene, keep in mind that UVs will be generated
+    for the first instance found. If the mesh is re-used with different scales
+    (and the scales are wildly different, more than half or twice), this will
+    result in inefficient lightmaps. Don't reuse a source mesh at significantly
+    different scales if you are planning to use lightmapping.
```

Comment:
It would be good to mention `GeometryInstance.lightmap_scale` here, since it can be used to give more lightmap resolution to certain instance in the scene.